bats/fb-test-foreman.bats

```diff
@@ -10,7 +10,8 @@ set -o pipefail
 }
 
 @test "check web app is up" {
-  curl -sk "https://localhost$URL_PREFIX/users/login" | grep login-form
+  VERSION=`curl -sk "https://localhost$URL_PREFIX/status" | jq .version`
+  ! grep -v -H -x "$FOREMAN_EXPECTED_VERSION" "$VERSION"
```

Comment:
how about this instead?

Reply:
Interesting approach, i completely forgot this endpoint even exists. Only issue is that in nightlies we don't have `FOREMAN_EXPECTED_VERSION` set iirc, so it would fail to find it.  Perhaps check `status` is `ok` instead since we check the version already earlier?

Reply:
@tbrisker updated.